src/main/java/com/openshift3/client/model/build/BuildTriggerType.java

```diff
@@ -0,0 +1,15 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors: Red Hat, Inc.
+ ******************************************************************************/
+package com.openshift3.client.model.build;
+
+public enum BuildTriggerType {
+	generic,
```

Comment:
@adietish we are not certain others will ever be introduced. I would say places where we parse for the enum we should probably have error handling code, log, and return null or something.  The possible value should be known to exist within a specific version of the api.